packages/block-library/src/post-author/edit.js

```diff
@@ -30,6 +31,14 @@ const AUTHORS_QUERY = {
 	per_page: 100,
 };
 
+const DEFAULT_AVATAR_SIZES = [ 24, 48, 96 ].map( ( size ) => ( {
+	value: size,
+	label: `${ size } x ${ size }`,
+} ) );
```

Comment:
The default value is hard-coded, but the avatar size retrieved by the REST API [has a filter hook](https://github.com/WordPress/wordpress-develop/blob/d9ec8536f7961ca066dc54c39e53fd41cbc5ecab/src/wp-includes/rest-api.php#L1242).

Therefore, if the consumer has changed the size variation on the hook, it may not match the default variation. However, there does not appear to be an endpoint to directly retrieve the result of the rest_get_avatar_sizes() function.

If you have a better way to do this, please let me know.

Reply:
I think that the more similar the blocks are, the easier it will be for the users.